accord-core/src/main/java/accord/impl/SimpleProgressLog.java

```diff
@@ -129,6 +131,7 @@ boolean isAtLeast(GlobalStatus equalOrGreaterThan)
         Set<Id> globalNotPersisted;
         GlobalPendingDurable globalPendingDurable;
 
+        Object debugInvestigating;
```

Comment:
I'm assuming this can be removed?

Reply:
I'd prefer to keep it, it can be helpful for debugging issues thrown up by the burn test. This isn't going to be a production implementation after all.